### DISCUSSION TITLE
Only index.html is showing on the GitHub Pages

### DISCUSSION THREAD

**Discussion by SanKukharchuk** [community/community#55151] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there everyone! I am just learning git and this is my second project. I faced the problem with hosting my project on GitHub Pages.
I checked my codes and possible reasons, the project works fine locally and I'm stuck ._.

At first I encountered the error that only repository name showing on the public link created by Github. I moved the index.html to root folder and changed the path in the webpack config. And now only index.html renders on the GitHub Pages. And I see 404 error in the console. I thought that there was an error in the webpack settings, which I configured for this project for the first time. Сould there be a problem with the path to index.js or with caching? I cant figure it out. 

I have no idea what do now 🙁. Could anyone be so kind to help me with this? Any help will be heartily ❤appreciated.

https://github.com/SanKukharchuk/britlex-landing

[
![zzvs](https://github.com/community/community/assets/119688675/43b7eea6-628e-4a64-8afc-bf267e216397)
](https://github.com/SanKukharchuk/britlex-landing)

**Comment by hrishikesh-kocharekar-dev** [comment]


**Reply by SanKukharchuk** [reply, marked as answer]
This solution helped me today:
1 git checkout -b gh-pages
2 git push
3 npm run deploy
4 in the GitHub settings change the root folder to gh-pages
Everything works fine now, thanks for your effort to help me figure it out!

**Reply by Sai1974** [reply]
Where should we run the above commands

**Reply by LiteBrite82** [reply]
those all get run in the terminal

**Reply by pawanongithub** [reply]
i have done everything and also got live project  ....but the issue is i get index page only and other 3 pages are not working and showing error 404.....what to do...
